usaspending_api/awards/tests/test_awards_idv_v2.py

```diff
@@ -0,0 +1,300 @@
+    mommy.make("awards.TransactionNormalized", **trans_cont)
+    mommy.make("references.Agency", **ag)
+    mommy.make("references.LegalEntity", **le)
+
+    asst_data = {
+        "pk": 1,
+        "transaction": TransactionNormalized.objects.get(pk=1),
+        "cfda_number": 1234,
+        "cfda_title": "farms",
+    }
+
+    latest_transaction_contract_data = {
+        "pk": 2,
+        "transaction": TransactionNormalized.objects.get(pk=2),
+        "type_of_contract_pric_desc": "FIRM FIXED PRICE",
+        "naics": "333911",
+        "naics_description": "PUMP AND PUMPING EQUIPMENT MANUFACTURING",
+        "idv_type_description": "IDC",
+        "type_of_idc_description": "INDEFINITE DELIVERY / INDEFINITE QUANTITY",
+        "multiple_or_single_aw_desc": "MULTIPLE AWARD",
+        "dod_claimant_program_code": "C9E",
+        "clinger_cohen_act_pla_desc": "NO",
+        "commercial_item_acquisitio": "A",
+        "commercial_item_test_desc": "NO",
+        "consolidated_contract_desc": "NOT CONSOLIDATED",
+        "cost_or_pricing_data_desc": "NO",
+        "construction_wage_rat_desc": "NO",
+        "evaluated_preference_desc": "NO PREFERENCE USED",
+        "extent_competed": "D",
+        "fed_biz_opps_description": "YES",
+        "foreign_funding_desc": "NOT APPLICABLE",
+        "information_technolog_desc": "NOT IT PRODUCTS OR SERVICES",
+        "interagency_contract_desc": "NOT APPLICABLE",
+        "major_program": None,
+        "purchase_card_as_paym_desc": "NO",
+        "multi_year_contract_desc": "NO",
+        "number_of_offers_received": None,
+        "price_evaluation_adjustmen": None,
+        "product_or_service_code": "4730",
+        "program_acronym": None,
+        "other_than_full_and_o_desc": None,
+        "sea_transportation_desc": "NO",
+        "labor_standards_descrip": "NO",
+        "small_business_competitive": "False",
+        "solicitation_identifier": None,
+        "solicitation_procedures": "NP",
+        "fair_opportunity_limi_desc": None,
+        "subcontracting_plan": "B",
+        "program_system_or_equipmen": "000",
+        "type_set_aside_description": None,
+        "materials_supplies_descrip": "NO",
+        "domestic_or_foreign_e_desc": "U.S. OWNED BUSINESS",
+        "ordering_period_end_date": "2025-06-30",
+        "period_of_performance_star": "2010-09-23",
+        "last_modified": "2018-08-24",
+        "agency_id": "192",
+        "referenced_idv_agency_iden": "168",
+        "piid": "0",
+        "parent_award_id": "1",
+    }
+    mommy.make("awards.TransactionFABS", **asst_data)
+    mommy.make("awards.TransactionFPDS", **latest_transaction_contract_data)
+    award_1_model = {
+        "pk": 1,
+        "type": "IDV_B_B",
+        "category": "idv",
+        "piid": 1234,
+        "type_description": "INDEFINITE DELIVERY / INDEFINITE QUANTITY",
+        "description": "lorem ipsum",
+        "generated_unique_award_id": "ASST_AW_3620_-NONE-_1830212.0481163",
```

Comment:
IDV GAUIs start with "CONT_AW"... not an essential change, but this could confuse someone.